tests/common/test_cstype_conversion.py

```
@@ -43,6 +43,20 @@ def test_auto_conversion_from_double_to_float(self):
         assert isinstance(cs_double, float)
         assert 1996.1012 == cs_double
 
+    def test_quote_constructor_retains_timezone(self):
+        dt = datetime.fromisoformat('2000-03-26 23:00+0000')
+        q = Quote(
```

Ruff (0.12.2)

Undefined name `Quote`

(F821)